tasks/modules/logging/syslog.rb

```diff
@@ -2,9 +2,35 @@
 
 module Logging
   module Syslog
-    def self.log message
+    def self.log message, level = nil
+
+      # Tumbler Collin severity mapping:
+      # - http://www.rubydoc.info/gems/collins_client/0.2.17/Collins/Api/Logging/Severity
+      #
+      # Compared to Syslog, Collins has an additional level 'note', for an operator input.
+      # We should not come into this level via Genesis.
+      # Anyway, handle it via the default level.
+      priority = case level
```

Comment:
I'm not sure I agree (i.e. I'm OK with using symbols but see strings as OK too).  If you stick with strings the matching should be anchored with ^ at least.  i.e. I'm OK with EMERGENCY being treated the same as EMER or EMERG.
OTOH there is something to be said for consistency, like being able to search the code easier.